pkg/apis/pipeline/v1alpha1/cloud_event_resource.go

```diff
@@ -0,0 +1,101 @@
+/*
+Copyright 2019 The Knative Authors.
```

Comment:
Nit: Use Tekton authors instead of Knative. See https://github.com/tektoncd/pipeline/pull/1089

Reply:
ups, nice catch, thank you!